controllers/velero_test.go

```diff
@@ -4108,6 +4109,426 @@ func TestDPAReconciler_buildVeleroDeployment(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "given wesley DPA CR for minio s3, appropriate Velero Deployment is built",
+			veleroDeployment: &appsv1.Deployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-velero-deployment",
+					Namespace: "test-ns",
+				},
+				Spec: appsv1.DeploymentSpec{
+					Selector: &metav1.LabelSelector{MatchLabels: veleroDeploymentMatchLabels},
+				},
+			},
+			dpa: func() *oadpv1alpha1.DataProtectionApplication {
+				dpa := &oadpv1alpha1.DataProtectionApplication{}
+				err := yaml.Unmarshal([]byte(`apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
```

Comment:
can Go struct be used instead of string literal :question:

Reply:
I was just too lazy to covert. I think I'll close it later